src/components/organisms/SectionQuotes.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { ReactElement, useEffect, useRef, useState } from 'react'
+import styles from './SectionQuotes.module.css'
+import { graphql, useStaticQuery } from 'gatsby'
+import { nanoid } from 'nanoid'
+import classNames from 'classnames/bind'
+import Container from '../atoms/Container'
+
+const cx = classNames.bind(styles)
+
+const query = graphql`
+  query QuotesQuery {
+    file(relativePath: { eq: "quotes.json" }) {
+      childContentJson {
+        title
+        quotes {
+          name
+          profilePicture
+          quote
+        }
+      }
+    }
+    profilePictures: allFile(
```

Comment:
issue(blocking): One problem with this approach is that the name of the file as well as the `profilePicture` in the `quotes.json` always need to be synced - without a clear link for any *non-coding contributor*. Meaning the link is made here in code, and not in the `.json` file.

I think in the siteMetadata a similar issue was solved pretty smoothly. Have a look at the `siteImage` in `site.json`. This is then accessed in the `hooks/useSiteMetadata.ts` hook.
Basically, given a valid filepath, the string is automatically mapped to a file (image in this case) and the necessary `childImageSharp` props can be used as below.

Would be a nice solution, if this can be applied here, too.